src/theme/src/hooks/useInputAppearance.js

```diff
@@ -17,7 +18,9 @@ const hoverPlaceholderState = '&:hover::placeholder'
 const focusPlaceholderState = '&:focus::placeholder'
 
 const getInputStyles = (theme, appearance) => {
-  const { inputs } = theme
+  const { inputs: themeStyles } = theme
+  const defaultStyles = getDefaultStyles(theme)
+  const inputStyles = { ...defaultStyles, ...themeStyles }
```

Comment:
Hm... IMO we should either:
1. explicitly tell consumers that if they don't pass in a complete theme object we will merge it with the v6 theme
2. provide a `createTheme` method for users to pass partial theme objects